### DISCUSSION TITLE
panic: unexpected error: fork/exec /usr/bin/sh: no such file or directory - macos

### DISCUSSION THREAD

**Discussion by berkayersengun** [helmfile/helmfile#699] [Questions] [answered]
Hi, I am trying to use helmfile client on macos (13.1). But whenever I try to run helmfile commands, I am getting the error below.
I also tried on ubuntu, had the similar issue. On ubuntu, i can create a symlink for  /usr/bin/sh and fix the issue but in macos the filesystem is readonly and not possible the create a symlink or copy the sh to the location  /usr/bin/sh to fix this error.

Is there any parameter or configuration to help fixing this issue ? maybe providing an alternative path to sh binary? 

helmfile version: v0.150.0

```
panic: unexpected error: fork/exec /usr/bin/sh: no such file or directory

goroutine 69 [running]:
github.com/helmfile/helmfile/pkg/helmexec.Output(0xc000b33ce0, {0x0, 0x0, 0xc0005829c0?})
	github.com/helmfile/helmfile/pkg/helmexec/runner.go:107 +0xda5
github.com/helmfile/helmfile/pkg/tmpl.(*Context).EnvExec.func2()
	github.com/helmfile/helmfile/pkg/tmpl/context_funcs.go:177 +0x2d
golang.org/x/sync/errgroup.(*Group).Go.func1()
	golang.org/x/sync@v0.0.0-20220722155255-886fb9371eb4/errgroup/errgroup.go:75 +0x64
created by golang.org/x/sync/errgroup.(*Group).Go
	golang.org/x/sync@v0.0.0-20220722155255-886fb9371eb4/errgroup/errgroup.go:72 +0xa5
```

**Comment by yxxhero** [comment]
Please add some debug logs by adding  ——debug flag. @berkayersengun

**Comment by berkayersengun** [comment, marked as answer]
Hi @yxxhero ,
I found the cause, for this error. this was a chart related issue not helmfile cli itself. Apparently, there is an exec command in one of templates and looking for sh at a certain path and failing to render. Sorry for that.
